testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -10815,3 +10815,448 @@ fn injected_signatures_are_ignored_across_boundaries() {
 
     assert!(new_spawned_signer.stop().is_none());
 }
+
+/// Test a scenario where:
+/// Two miners boot to Nakamoto.
+/// Sortition occurs. Miner 1 wins.
+/// Miner 1 proposes a block N
+/// Signers accept and the stacks tip advances to N
+/// Miner 1's block commits are paused so it cannot confirm the next tenure.
+/// Sortition occurs. Miner 2 wins.
+/// Miner 2 proposes block N+1
+/// Signers accept and the stacks tip advances to N+1
+/// Sortition occurs quickly, within first_proposal_burn_block_timing_secs. Miner 1 wins.
+/// Miner 1 proposes block N+1'
+/// Signers approve N+1', saying "Miner is not building off of most recent tenure. A tenure they
+///   reorg has already mined blocks, but the block was poorly timed, allowing the reorg."
+/// Miner 1 proposes N+2 and it is accepted.
+/// Asserts:
+/// - N+1 is signed and broadcasted
+/// - N+1' is signed and broadcasted
+/// - The tip advances to N+1 (Signed by Miner 1)
+/// - The tip advances to N+2 (Signed by Miner 1)
```

Comment:
I think the docstring here may be missing a Stacks block N+3?

Reply:
Updated.